src/main/flight/pid.c

```diff
@@ -445,12 +445,9 @@ void pidInitConfig(const pidProfile_t *pidProfile)
     horizonFactorRatio = (100 - pidProfile->horizon_tilt_effect) * 0.01f;
     maxVelocity[FD_ROLL] = maxVelocity[FD_PITCH] = pidProfile->rateAccelLimit * 100 * dT;
     maxVelocity[FD_YAW] = pidProfile->yawRateAccelLimit * 100 * dT;
-    ITermWindupPointInv = 0.0f;
     if (pidProfile->itermWindupPointPercent < 100) {
-        float ITermWindupPoint = (float)pidProfile->itermWindupPointPercent / 100.0f;
+        const float ITermWindupPoint = pidProfile->itermWindupPointPercent / 100.0f;
         ITermWindupPointInv = 1.0f / (1.0f - ITermWindupPoint);
-    } else {
```

Comment:
I don't think the `else` clause here can be dropped - `pidInitConfig()` is called at run time, so `ITermWindupPointInv` isn't guaranteed to be set to 1 at this point.

Also, n.b. it should really be `iTermWindupPoint` and `iTermWindupPointInv`.

Reply:
`static FAST_RAM float ITermWindupPointInv = 1.0f;`

https://github.com/betaflight/betaflight/blob/d9719ef3644ee7a58bd24276e606e49c404df017/src/main/flight/pid.c#L373

Reply:
This is the initial value. But if `pidInitConfig()` is called this is overwritten if `pidProfile->itermWindupPointPercent < 100`. If it is called again with `pidProfile->itermWindupPointPercent == 100`, the previous value will be stay in `ITermWindupPointInv`.

Reply:
Thank you.  Very clear.  Added back the else.

Reply:
One minor query.  Now the 'else' is back, iTermWindupPointInv is set to a specific value every loop...  which is more efficient, the current:
```
static FAST_RAM float iTermWindupPointInv = 1.0f;
```
or the original
```
static FAST_RAM_ZERO_INIT float iTermWindupPointInv;
```

Reply:
I'n positive we have more than one case when we rely on the init value like that, and we always assumed that FC and subsystem initialization isn't called more than once.
Are we now going to start hunting for other places then?

Reply:
I think we do need to distinguish between functions that are only called in `init.c`, and others that are called at runtime as well. `pidInitConfig()` is called in places like `rc_adjustments.c`, so we cannot rely on the static initialiser values to be still in place when it is called.